src/components/featurescomponent.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+// Not sure where to pass in feature state data... this is just a place holder
+//import {FEATURES} from './features';
+
+function Features({FEATURES}) {
+    return(
+        <>
+            <div class="intro">
+                <h2 class="text-center">Features </h2>
+                <p class="text-center">Nunc luctus in metus eget fringilla. Aliquam sed justo ligula. Vestibulum nibh erat, pellentesque ut laoreet vitae.</p>
+            </div>
+            {/* dynamic rendering of feature cards as displayed on JOEPRO starter site */}
+            <div class="row justify-content-center features">
+                {FEATURES.map(feature => {
```

Comment:
the below JSX you wrote is something that could potentially be re-used throughout the project.

So lets make that into a re-usable <Card /> component.

This 'Feature' component however is something that will only show up on the Landing page. I'm sure you just made it a Component because my template code wasn't up, but this section should just be added into the `Home` page in the `Pages` directory